test/rubocop/cop/minitest/assert_raises_compound_body_test.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AssertRaisesCompoundBodyTest < Minitest::Test
+  def test_registers_offense_when_multi_statement_bodies
+    assert_offense(<<~RUBY)
+      assert_raises(MyError) do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^ Reduce `#assert_raises` body to contain only the raising code.
```

Comment:
Can you tweak the message?

```suggestion
      ^^^^^^^^^^^^^^^^^^^^^^^^^ Reduce `assert_raises` block body to contain only the raising code.
```